commands/channel.go

```diff
@@ -83,6 +85,17 @@ Channel can be specified by [team]:[channel]. ie. myteam:mychannel or by channel
 	RunE:    makeChannelPrivateCmdF,
 }
 
+var SearchChannelCmd = &cobra.Command{
+	Use:   "search [channel]\n  mattermost search --team [team] [channel]",
```

Comment:
it should be `mmctl` not `mattermost`, right?

Reply:
Fiexd!